### PR TITLE
docs: node export with images

### PR DESCRIPTION
**Pull request by aretakisv**
self explaining, could be used for our users when posting an issue!

### CODE REVIEW

**Comment by robertsLando** [comment]
@billiaz This is so much useful! I would also explain to also attach logs both from zwavejs and zwavejs2mqtt

File: docs/troubleshooting/node_export.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Reporting a Node Bug
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:1]: I would be more generic here, `Report a bug`

File: docs/troubleshooting/node_export.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+When it comes to node issues, it is best to export specific information, which can help us debug and help with the problem
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:3]: Also adding logs from `zwavejs` and zwavejs2mqtt` would be helpful

File: docs/troubleshooting/node_export.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Exporting node details
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:5]: I would use just one title `Export node json`  and then create `###` for sub steps

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/node_export.md (hunk 3 above)
Comment (billiaz) [reply docs/troubleshooting/node_export.md:5]: Done! :)

File: docs/troubleshooting/node_export.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Report a Node Bug
+
+When it comes to node issues, it is best to export specific information, which can help us debug and help with the problem
+
+## Export node json
+
+Each node hass an Export button, this will export all data of this specific node. To perform this export, use our screen shots and steps as your guide.
+
+### Select node from your webinterface
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:9]: I prefer to use a numbered list here:

1. Select node from web interfcace
....
2.
....

File: docs/troubleshooting/node_export.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Selecting the node as showing in the image.
+
+![Selected node](../_images/troubleshoot_node_select.png)
+
+Will result showing you more details of the node at the bottom of your interface.
+
+![Export location](../_images/troubleshoot_export.png)
+
+### export Node json
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:19]: Duplicated title, fixed by using list instead

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/node_export.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+2.  Export Node json by clicking on export shown on screenshot! this will generate a json of this specific node.
+
+    ![Selected node](../_images/troubleshoot_export_2.png)
+
+3.  Grab the file and attach it on your github Issue
+
+    ![Grab file](../_images/troubleshoot_node_json.png)
+    
+## Simple as that!
```
Comment (robertsLando) [inline docs/troubleshooting/node_export.md:24]: Remove this

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Review by robertsLando — APPROVED** [review]
LGTM

**Pull request merged**